### DISCUSSION TITLE
Get Multiple Properties in One Query?

### DISCUSSION THREAD

**Discussion by keeper2597** [leolabs/ableton-js#73] [Q&A] [answered]
Is it possible to gain two or more properties on a single get request somehow?

e.g. `clip.get(["signature_numerator", "signature_denominator"])`

Or do you have to make requests for each trait?

Thanks!

**Comment by leolabs** [comment, marked as answer]
I usually just use `Promise.all` for this:

```ts
const [numerator, denominator] = await Promise.all([
  clip.get("signature_numerator"),
  clip.get("signature_denominator"),
]);
```

**Reply by keeper2597** [reply]
Cool.  Makes sense, thanks!
